src/plugins/data/public/ui/apply_filters/apply_filter_popover_content.tsx

```diff
@@ -64,7 +64,8 @@ export default class ApplyFiltersPopoverContent extends Component<Props, State>
       <EuiForm>
         {mappedFilters.map((filter, i) => (
           <EuiFormRow key={i}>
-            <EuiSwitch
+            <EuiCheckbox
```

Comment:
I think ideally we should use [EuiCheckboxGroup](https://eui.elastic.co/#/forms/form-controls#checkbox-group) here and get nicer spacings. 

But I think this is already a nice improvement and we can merge as is